.travis.yml

```diff
@@ -1 +1,4 @@
 language: java
+
```

Comment:
are we going to use Travis?

Reply:
We kept it on Travis at the moment so as to not include too much in the PR.